tests/Dockerfile.rocky-8

```diff
@@ -0,0 +1,199 @@
+#
+# Sysbox Test Container Dockerfile (Rocky-8 image)
+#
+# This Dockerfile creates the sysbox test container image. The image
+# contains all dependencies needed to build, run, and test sysbox.
+#
+# The image does not contain sysbox itself; the sysbox repo
+# must be bind mounted into the image. It can then be built,
+# installed, and executed within the container.
+#
+# The image must be run as a privileged container (i.e., docker run --privileged ...)
+# Refer to the sysbox Makefile test targets.
+#
+# This Dockerfile is based on a similar Dockerfile in the OCI runc
+# github repo, but adapted to sysbox testing.
+#
+# Instructions:
+#
+# docker build -t sysbox-test .
+#
+
+FROM rockylinux
```

Comment:
Please set an explicit tag here (e.g. "rockylinux:8.5") to ensure that we don't break anything when bringing a new "latest" release.

Reply:
Makes sense to change that. Should we do the change in the tests/Dockerfile.almalinux-8 file accordingly ?

Reply:
Oh we missed that. Yes please, do the same in the almalinux file too. Thanks!